### PR TITLE
Increasing file descriptor limit (OPS-728)

### PR DESCRIPTION
**Pull request by bilalahmad99**

### CODE REVIEW

File: playbooks/roles/locust/tasks/main.yml [hunk 1]
```diff
@@ -72,3 +72,9 @@
     name={{ locust_service_name }}
   when: not disable_edx_services
   sudo_user: "{{ supervisor_service_user }}"
+
+- name: increase file descriptor limit of the system (Session Logout and Login would be required)
+  lineinfile:
+    dest: "{{ ulimit_conf_file }}"
```
Comment (arbabnazar) [inline playbooks/roles/locust/tasks/main.yml:78]: what's the value of `ulimit_con_file`?

Comment (bilalahmad99) [inline playbooks/roles/locust/tasks/main.yml]: missed it

Comment (arbabnazar) [inline playbooks/roles/locust/tasks/main.yml:78]: Can you please share the verification method? I think we need to reboot to system after configuring these value to take affect.

Comment (bilalahmad99) [inline playbooks/roles/locust/tasks/main.yml:78]: No, just a logout of the session and log back in would be required. I verified it on my sandbox using command "ulimit -n".

**Comment by e0d** [comment]
:+1:

**Pull request merged**